apps/codeforafrica/src/components/NavBarNavList/NavBarNavList.js

```diff
@@ -4,11 +4,25 @@ import { SvgIcon } from "@mui/material";
 import PropTypes from "prop-types";
 import React from "react";
 
+import FacebookIcon from "@/codeforafrica/assets/icons/Type=facebook, Size=24, Color=CurrentColor.svg";
+import GitHubIcon from "@/codeforafrica/assets/icons/Type=github, Size=24, Color=CurrentColor.svg";
+import InstagramIcon from "@/codeforafrica/assets/icons/Type=instagram, Size=24, Color=CurrentColor.svg";
+import LinkedInIcon from "@/codeforafrica/assets/icons/Type=linkedin, Size=24, Color=CurrentColor.svg";
+import SlackIcon from "@/codeforafrica/assets/icons/Type=slack, Size=24, Color=CurrentColor.svg";
 import TwitterIcon from "@/codeforafrica/assets/icons/Type=twitter, Size=24, Color=CurrentColor.svg";
 import NavListItem from "@/codeforafrica/components/NavListItem";
 
+const mapPlatformToIcon = {
```

Comment:
```suggestion
const platformToIconMap = {
```

`mapXXX` is cool if this were a `function`.